### ISSUE TITLE
`java_tools` `github_url` is incorrect

### ISSUE THREAD

**Issue by brentleyjones** [bazelbuild/rules_java#193]
https://github.com/bazelbuild/rules_java/blob/3a3a98f5bfbcdb1d4185d5f98b708d3459b0ec1f/java/repositories.bzl#L49

should be https://github.com/bazelbuild/java_tools/releases/download/java_v13.6/java_tools-v13.6.0.zip

Found on a unrelated error:
```
(12:55:11) WARNING: Download from https://mirror.bazel.build/bazel_java_tools/releases/java/v13.6.0/java_tools-v13.6.0.zip failed: class com.google.devtools.build.lib.bazel.repository.downloader.ContentLengthMismatchException Bytes read 5767136 but wanted 37662433
(12:55:11) WARNING: Download from https://github.com/bazelbuild/java_tools/releases/download/java_v13.6.0/java_tools-v13.6.0.zip failed: class java.io.FileNotFoundException GET returned 404 Not Found
(12:55:11) ERROR: An error occurred during the fetch of repository 'rules_java~~toolchains~remote_java_tools':
   Traceback (most recent call last):
	File "/private/var/tmp/_bazel_vagrant/44b08d771321dac1bc850a6d84c72843/external/bazel_tools/tools/build_defs/repo/http.bzl", line 131, column 45, in _http_archive_impl
		download_info = ctx.download_and_extract(
Error in download_and_extract: java.io.IOException: Error downloading [https://mirror.bazel.build/bazel_java_tools/releases/java/v13.6.0/java_tools-v13.6.0.zip, https://github.com/bazelbuild/java_tools/releases/download/java_v13.6.0/java_tools-v13.6.0.zip]
```

**Comment by fmeum** [comment]
@hvadehra

**Comment by hvadehra** [comment]
Fixed in `v7.6.4`, publishing to BCR pending: https://github.com/bazelbuild/bazel-central-registry/pull/2276
